### DISCUSSION TITLE
Plugins Photoshop / Illustrator

### DISCUSSION THREAD

**Discussion by heliomsolivas** [frontendbr/forum#102] [Perguntas e Respostas]
Pessoal não sei com vocês mas eu converto muito .psd / .ai aqui onde trabalho, queria saber se vocês utilizam algum plugin pra ajudar no front-end, eu usava o css3ps que pelo menos para saber os valores dos box-shadows ele facilitava muito. Eu estava atrás de algo principalmente pra ajudar a achar os paddings/font-sizes mais rapidamente.

**Comment by vitoravale** [comment]
Eu costumava utilizar muito o CSS Hat ([https://csshat.com/](url)), é um plugin para o Photoshop que permite extrair todas as informações de uma layer/elemento incluindo cores, gradientes, medidas, bordas, entre outros. E o bacana é que ele já sugere a importação de fontes do Typekit e Google Fonts.

**Comment by twobanks** [comment]
Utilizo o [css3ps](http://css3ps.com/), ele quebra um galho gigantesco.

**Comment by diessica** [comment]
Veja o [Avocode](https://avocode.com/).

**Comment by heliomsolivas** [comment]
@diessica eu tentei usar o Avocode aqui, mas me faltava letter-spacing e mais alguma coisa que tive que voltar pro photoshop :(

**Comment by Marcelosilva10** [comment]
http://www.webbsy.com/ único prolema não é free.

**Comment by vicainelli** [comment]
Quando eu trabalhava em agência, a intensidade de `.psd` era muito alta, usava o [CSS Hat](http://csshat.com/). 

Hoje, um pouco mais tranquilo, trabalhando com poucos produtos, gosto de usar o [SketchApp](https://www.sketchapp.com/), e deixo o Photoshop mais pra aquele trabalho chato de tratar dezenas de imagens usando actions.

> Eles dão 50% de desconto pra estudantes, é só mandar email com a carteirinha que eles mandam a fatura.
